live-build/misc/ansible-roles/appliance-build.dcenter/tasks/ldap.yml

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (c) 2018 by Delphix. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+---
+
+# Adapted from:
+#   * https://help.ubuntu.com/community/AutofsLDAP
+#   * https://www.digitalocean.com/community/tutorials/how-to-authenticate-client-computers-using-ldap-on-an-ubuntu-12-04-vps
+
+- name: Ubuntu Linux LDAP Access | Install LDAP OS Packages
+  apt:
+    name={{ item }}
```

Comment:
please reformat module fields here and below to use `:` instead of `=`.. e.g.
```
apt:
  name: "{{ item }}"
  state: present
```
this way we have a consistent style across all the ansible roles.